# HG changeset patch
# User Dave Parks <davep@lindenlab.com>
# Date 1308673064 18000
# Node ID 95c5639a3f80920e8dc54703d894517dd7694edf
# Parent  6af10678de4736222b2c3f7e010e984fb5b327de
SH-208 Disable VBO on all intel graphics chips (stability improvement).

diff -r 6af10678de47 -r 95c5639a3f80 indra/newview/featuretable.txt
--- a/indra/newview/featuretable.txt	Mon Jun 20 16:42:31 2011 -0700
+++ b/indra/newview/featuretable.txt	Tue Jun 21 11:17:44 2011 -0500
@@ -1,4 +1,4 @@
-version 29
+version 30
 
 // NOTE: This is mostly identical to featuretable_mac.txt with a few differences
 // Should be combined into one table
@@ -297,6 +297,7 @@
 
 list Intel
 RenderAnisotropic			1	0
+RenderVBOEnable				1	0
 
 list GeForce2
 RenderAnisotropic			1	0
diff -r 6af10678de47 -r 95c5639a3f80 indra/newview/featuretable_xp.txt
--- a/indra/newview/featuretable_xp.txt	Mon Jun 20 16:42:31 2011 -0700
+++ b/indra/newview/featuretable_xp.txt	Tue Jun 21 11:17:44 2011 -0500
@@ -1,4 +1,4 @@
-version 29
+version 30
 
 // NOTE: This is mostly identical to featuretable_mac.txt with a few differences
 // Should be combined into one table
@@ -295,6 +295,7 @@
 
 list Intel
 RenderAnisotropic			1	0
+RenderVBOEnable				1	0
 
 list GeForce2
 RenderAnisotropic			1	0
